pkg/lightning/checkpoints/checkpoints.go

```diff
@@ -106,6 +106,7 @@ const (
 			table_name varchar(261) NOT NULL,
 			engine_id int NOT NULL,
 			status tinyint unsigned DEFAULT 30,
+			commit_ts bigint unsigned NOT NULL DEFAULT 0,
```

Comment:
Could you add a test where commit_ts = uint64::MAX?

Reply:
I found it is not easy to test with uint64::MAX. It seems the [go-sqlmock](https://github.com/DATA-DOG/go-sqlmock) we used in test doesn't allow large uint64 value. It may be caused by golang/go#9373. I test that converting uint64 in ExecContext  to a string is ok. But I don't known if there is a potential risk. Do you have any suggestions?